src/components/data/listing/Listing.js

```diff
@@ -73,6 +73,8 @@ function Listing(props) {
         order,
         orderBy,
         onRouteToListing,
+        toolbarVisibility = true,
+        rowDotMenuVisibility = true,
```

Comment:
👍 
I couldn't think of a simple way to make these changes when you suggested this before the break.  This is nice!